pyproject.toml.jinja

```diff
@@ -36,6 +36,7 @@ dynamic = ["version"]
 
 [project.urls]
 "Bug Tracker" = "https://github.com/{{orgname}}/{{projectname}}/issues"
+"Documentation" = "https://{{orgname}}.github.io{% if projectname != 'scipp' %}/{{projectname}}{% endif %}"
```

Comment:
Why skip the orgname? This would only work for project names matching their orgname? Also, it does not match our docs deploy step, which deploys to a `gh_pages` *branch`, not a repo, so I think we should always have the `orgname` here.

Reply:
Because https://scipp.github.io/scipp does not lead anywhere?
Unless I misunderstood something?

Reply:
Because Scipp docs are not deployed to a `gh_pages` branch. But `copier_template` does.

You are right that it will break only if there is a `scipp` project in a different org.